ui/browser/views/browser.jsx

```diff
@@ -63,7 +63,7 @@ class BrowserWindow extends Component {
           {...{ pages, navBack, navForward, navRefresh, minimize, maximize,
             close, openMenu, onLocationChange, onLocationContextMenu,
             onLocationReset, bookmark, unbookmark, pageAreaVisible, ipcRenderer,
-            setPageAreaVisibility }} />
+            setPageAreaVisibility, navigateTo }} />
```

Comment:
This is getting more and more horrible. Why not have some kind of object coalescing all these props instead of passing them one by one?